bc_obps/registration/tests/endpoints/test_useroperator_endpoints.py

```diff
@@ -776,3 +832,87 @@ def test_put_user_operator_operator_malformed_data(self):
         )
 
         assert put_response.status_code == 422
+
+    def test_put_duplicates_not_allowed(self):
+        baker.make(
+            Operator, legal_name='Legal Name', bc_corporate_registry_number="aaa1234321", cra_business_number=55555
+        )
+
+        operator = baker.make(Operator, bc_corporate_registry_number="yyy1234321")
+
+        user_operator = baker.make(UserOperator, user=self.user, operator=operator, role=UserOperator.Roles.ADMIN)
+
+        # duplicate CRA business number
```

Comment:
The [bug](https://github.com/bcgov/cas-registration/pull/736) is that cra business number and bc corporate number are never added to the db, so these tests currently fail (I wrote them before I realized that, oops.) In the bug ticket, I wrote a note to either uncomment them out or delete them depending on how the bug is handled.